packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -588,13 +624,6 @@ export const useArbTokenBridge = (
     try {
       const erc20Bridger = await Erc20Bridger.fromProvider(l2.provider)
 
-      const provider = l2Signer.provider
-      const isSmartContractAddress =
-        provider && (await provider.getCode(String(erc20L1Address))).length < 2
-      if (isSmartContractAddress && !destinationAddress) {
```

Comment:
NIT: Where do we handle this case now ie. if it's an SC wallet and the destination address is not present? Through UX? I'd still keep this check for safety here. Thoughts?

Reply:
yeah we should probably add that back